.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -49,64 +58,133 @@ export interface Props {
+      if (!isTextSelected) {
+        linkRef.current.click(); /* 4 */
+      }
+    }
+    /* 5 */
+    return () => {
+      curr.removeEventListener('click', handleClick, false);
+    };
+  }, []);
+
+  const componentClassName = clsx(
+    styles['baseline-card'],
+    linkHref && styles['baseline-card--clickable'],
+    className,
+  );
   return (
-    <Card className={componentClassName} {...other}>
-      <CardHeader className={styles['baseline-card__label']}>
-        {label}
-      </CardHeader>
-      <CardBody className={styles['baseline-card__body']}>{body}</CardBody>
-      {metadata && (
-        <CardFooter className={styles['baseline-card__footer']}>
-          <table className={styles['baseline-card__table']}>
-            <tbody className={styles['baseline-card__table-body']}>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Score
-                </th>
-                <td>
-                  <Tag
-                    className={styles['baseline-card__score']}
-                    hasOutline
-                    text={metadata.score}
-                    variant={metadata.variant}
-                  />
-                </td>
-              </tr>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Attempts
-                </th>
-                <td className={styles['baseline-card__table-data']}>
-                  {metadata.attempts}
-                </td>
-              </tr>
-              <tr className={styles['baseline-card__table-row']}>
-                <th
-                  className={styles['baseline-card__table-label']}
-                  scope="row"
-                >
-                  Line Passes On
-                </th>
-                <td className={styles['baseline-card__table-data']}>
-                  {metadata.deadline}
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </CardFooter>
-      )}
-    </Card>
+    /**
+     * 1) Wrapping the Card component in a container for attaching cardRef
+     * to the card without adding a ref prop to <Card />
+     */
+
+    <div className={styles['baseline-card__container']} ref={cardRef}>
+      <Card className={componentClassName} {...other}>
+        <CardHeader className={styles['baseline-card__label']}>
+          {label}
+        </CardHeader>
+        <CardBody className={styles['baseline-card__body']}>
+          {body}
+          {` `}
+          {linkHref && (
+            <a
+              className={styles['baseline-card__link']}
+              href={linkHref}
+              ref={linkRef}
+              rel="noreferrer"
+              target="_blank"
```

Comment:
Does the link always need to open in a new tab? Should that be based on props? Maybe instead of having `linkText` and `linkHref` props, you could have `linkProps` so devs can add in the `rel` and `target` if they want.

Reply:
I like your approach, thanks!